packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -713,14 +713,14 @@ export const workflowsRouter = router({
         });
         addedSteps.forEach(async (step) => {
           if (step) {
-            const newStep = step;
+            const { senderName, ...newStep } = step;
             newStep.sender = getSender({
               action: newStep.action,
               sender: newStep.sender || null,
-              senderName: newStep.senderName,
+              senderName: senderName,
             });
             const createdStep = await ctx.prisma.workflowStep.create({
-              data: { ...step, numberVerificationPending: false },
+              data: { ...newStep, numberVerificationPending: false },
```

Comment:
excluded senderName from newStep as it is not part of prisma.workflowSte. This caused the issue that workflow actions couldn't be added to a workflow anymore